xml/deployment_prep_zseries_prep_i.xml

```diff
@@ -360,69 +348,67 @@
+       Identifier for the profile. Use the string specified when
        added the distribution.
       </para>
      </callout>
      <callout arearefs="cobbler-profile-os">
       <para>
        Operating system version. Distribution to which the profile should
-       apply. You must use the string specified with
+       apply. Use the string specified with
        <option>--name=<replaceable>IDENTIFIER</replaceable></option> in the
-       importing step here.
+       importing step.
       </para>
      </callout>
      <callout arearefs="cobbler-profile-meta">
       <para>
-       Option needed for templating Kickstart files. Not used for &suse;, so
-       set to an empty value as specified in the example.
+       Option required for templating Kickstart files. Since it is not used for
+       &suse;, leave it empty.
       </para>
      </callout>
      <callout arearefs="cobbler-profile-opts">
       <para>
-       Space-separated list of kernel parameters. Should include at least the
-       <literal>install</literal> parameter as shown in the example.
+       Space-separated list of kernel parameters. It must  include at least the
+       <literal>install</literal> parameter.
       </para>
      </callout>
     </calloutlist>
    </sect4>
    <sect4 xml:id="sec-prep-makeavail-zpxe-cobbler-system">
     <title>Adding Systems</title>
     <para>
-     The last step that is required is to add systems to the Cobbler server. A
-     system addition needs to be done for every IBM &zseries; guest that should
-     boot via zPXE. Guests are identified via their z/VM user ID (in the
-     following example, an ID called <quote>linux01</quote> is assumed). Note
-     that this ID needs to be a lowercase string. To add a system, run the
-     following command:
+     The last step is to add systems to the Cobbler server. This step must be
+     performed for every IBM &zseries; guest that should boot via zPXE. Guests
+     are identified by their z/VM user ID (in the following example, the ID
+     <quote>linux01</quote>). Note that the ID must be lowercase. To add a
+     system, run the following command:
     </para>
 <screen>cobbler system add --name=linux01 --hostname=linux01.example.com \
 --profile=<replaceable>IDENTIFIER</replaceable> --interface=qdio \
 --ip-address=&wsIVip; --subnet=&subnetII;.255 --netmask=&subnetmask; \
 --name-servers=&dnsip; --name-servers-search=&exampledomain; \
 --gateway=&routerintipII; --kopts="<replaceable>KERNEL_OPTIONS</replaceable>"</screen>
     <para>
-     With the <option>--kopts</option> option you can specify the kernel and
-     installation parameters you would normally specify in the parmfile. The
-     parameters are entered as a space-separated list in the form of
+     The <option>--kopts</option> option allows you to specify the kernel and
+     installation parameters that are usually specified in the parmfile. The
+     parameters are specified using the following format:
```

Comment:
As it is an instruction, I would use active voice here: 'Specify the parameters in the following format'.